shared/pkg/utils/image_helper/image_helper.go

```diff
@@ -146,20 +147,15 @@ func stripDockerMetaFromCommand(command string) string {
 	return ret
 }
 
-func getV1Image(imageName string, registryOptions *image.RegistryOptions) (containerregistry_v1.Image, error) {
-	source, location, err := image.DetectSource(imageName)
-	if err != nil {
-		return nil, fmt.Errorf("falied to detect source for image: %s. %v", imageName, err)
-	}
+func getV1Image(imageName string, registryOptions *image.RegistryOptions, localImage bool) (containerregistry_v1.Image, error) {
 	log.Debugf("pulling image info directly from registry image=%q", imageName)
```

Comment:
`log.Debugf("pulling image info directly from registry image=%q", imageName)` - command should be under the `default` switch case, no?

Reply:
yes